### DISCUSSION TITLE
Validation Errors

### DISCUSSION THREAD

**Discussion by bawbanksy** [joshhanley/livewire-autocomplete#28] [Help] [answered]
What would be the best way to accomplish showing validation errors? Very basic use case of "required" being set on the input.

**Comment by bawbanksy** [comment]
I have added-

`@error($attributes['name'] . '.selected.id')
    <span class="text-xs font-medium text-red-500">{{ $message }}</span>
@enderror
`

to input.blade.php 


and am running this validation rule()

`    public function rules()
    {
        return [
            'projects.selected.id' => 'required',
        ];
    }`
    
    this is giving the desired outcome provided the "name" attribute matches.

**Comment by joshhanley** [comment, marked as answer]
@bawbanksy I personally don't like to add errors into my input components, as I prefer to create small composable blade components.

So I would have like an input group component or something like that, which handles the error consistently.

```blade
<x-input-group name="my-prop" label="My Prop">
    <x-autocomplete ... />
</x-input-group>
```
